proxy/http/HttpSM.cc

```diff
@@ -593,6 +593,11 @@ HttpSM::state_read_client_request_header(int event, void *data)
   case VC_EVENT_READ_READY:
   case VC_EVENT_READ_COMPLETE:
     // More data to parse
+
+    // If there is still a post body underway, defer processing until post body is done
```

Comment:
I think this is in the wrong method.

Reply:
Oops!  problem with hand patching.  Updated PR.  Should be on correct method now.